Add metrics to JS getting started

svrnm opened the pull request:
The auto instrumentation for http/express in Node.JS provides some out of the box metrics. By initializing a metricReader & exporter they get print out in the getting started. 

I also updated the exporters file accordingly  

Preview: 
* https://deploy-preview-2576--opentelemetry.netlify.app/docs/instrumentation/js/getting-started/nodejs/
* https://deploy-preview-2576--opentelemetry.netlify.app/docs/instrumentation/js/exporters/

pichlermarc commented on content/en/docs/instrumentation/js/exporters.md:
```diff
@@ -45,6 +52,13 @@ const sdk = new opentelemetry.NodeSDK({
+    exporter: new OTLPMetricExporter({
+      url: '<your-otlp-endpoint>/v1/metrics', // url is optional and can be omitted - default is http://localhost:4318/v1/metrics
+      headers: {}, // an optional object containing custom headers to be sent with each request
+      concurrencyLimit: 1, // an optional limit on pending requests
```
```suggestion
```
We could leave this out as to be consistent with the trace exporter example above :slightly_smiling_face:

pichlermarc commented on content/en/docs/instrumentation/js/exporters.md:
```diff
@@ -18,25 +18,32 @@ Jaeger) you'll want to use an exporter package, such as
 } from "@opentelemetry/exporter-trace-otlp-http";
+import {
+  OTLPMetricExporter
+} from "@opentelemetry/exporter-metrics-otlp-http";
```
Above we recommend installing the gRPC exporter, but here we're using oltp/json. I wonder if this example should be using the `@opentelemetry/exporter-metrics-otlp-proto` and `@opentelemetry/exporter-trace-otlp-proto` packages instead as otlp/json is not really widely accepted.

svrnm replied:
Oh, that grpc/http thing was a mix up, I wanted to use http for both, because I wanted to align it with what is done for the OTLPTraceExporter.

I can change both to proto then?

svrnm replied:
done

pichlermarc commented on content/en/docs/instrumentation/js/exporters.md:
```diff
@@ -59,6 +73,9 @@ const {
 } = require("@opentelemetry/exporter-trace-otlp-http");
+const {
+  OTLPMetricExporter
+} = require("@opentelemetry/exporter-metrics-otlp-http");
```
same here with the exporter package :slightly_smiling_face:

svrnm replied:
done

pichlermarc reviewed (commented):
Thank you for adding this. Just a few notes about the exporters/exporter-setup :slightly_smiling_face:

chalin commented on content/en/docs/instrumentation/js/getting-started/nodejs.md:
```diff
@@ -4,7 +4,7 @@ aliases: [/docs/js/getting_started/nodejs]
 ---
 
-This guide will show you how to get started with tracing in Node.js.
+This guide will show you how to get started with OpenTelemetry in Node.js.
```
```suggestion
This page will show you how to get started with OpenTelemetry in Node.js.
```

chalin reviewed (approved):
Only one nit, see inline suggestion.

svrnm commented:
> Thank you for adding this. Just a few notes about the exporters/exporter-setup 🙂

Updated to proto, please take another look, I hope I fixed all of them :-)

pichlermarc reviewed (approved):
Looks great :slightly_smiling_face:
